src/block_cache/block_chunkcache.c

```diff
@@ -1032,6 +1129,25 @@ __wt_chunkcache_teardown(WT_SESSION_IMPL *session)
     return (ret);
 }
 
+/*
+ * __wt_chunkcache_salvage --
+ *     Remove any knowledge of any extant chunk cache metadata. We can always rebuild the cache
+ *     later, so make no attempt at a "real" salvage.
+ */
+int
+__wt_chunkcache_salvage(WT_SESSION_IMPL *session)
+{
+    WT_DECL_RET;
+
+    __wt_spin_lock(session, &S2C(session)->schema_lock);
+    ret = __wt_schema_drop(session, WT_CC_URI, NULL);
```

Comment:
This bit of code needs a little help - it seems there are ways for this function to get called while holding the schema lock. @ddanderson do you think it'd be sufficient to check if we're holding the schema lock and then acquire it? Or am I missing something bigger?

Reply:
Done in 6435bb6, thanks for the advice!